sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatClientTest.java

```diff
@@ -41,6 +44,7 @@
     @Override
     protected void beforeTest() {
```

Comment:
do we have a base test class for java?  if so, can we add the logic into the base class so that we don't need to repeat the logic every time

Reply:
Addressed! Thanks Rong